AWSPolly/AWSPollySynthesizeSpeechURLBuilder.m

```diff
@@ -291,120 +293,122 @@ - (NSString *)stringFromLanguageCode: (AWSPollyLanguageCode)languageCode {
 
 - (NSString *)stringFromVoiceId:(AWSPollyVoiceId)voiceId{
```

Comment:
Can we use NSValueTransformer from [voiceIdJSONTransformer](https://github.com/aws-amplify/aws-sdk-ios/blob/fe61cd1882a0969617e3286fa34e21a05133ac28/AWSPolly/AWSPollyModel.m#L800) to get the string from the AWSPollyVoiceId? So that avoiding changing at two places.

Reply:
Great suggestion. Fixed.